### PR TITLE
Adding in translation for Work show page for identifiers

### PR DESCRIPTION
**Pull request by carolyncole**
refs #1176 

Original view:
screen shot 2018-03-14 at 12 44 44 pm

New view:
screen shot 2018-03-14 at 12 43 51 pm

### CODE REVIEW

File: app/renderers/translated_doi_attribute_renderer.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Override CurationConcerns::Renderers::FacetedAttributeRenderer so we can pass different values
+# for facets. This allows us to display a faceted search link using the text that the user
+# originally entered while using the normalized text that was created when the facet was
+# "cleaned" using the SolrDocumentGroomer.
```
Comment (awead) [inline app/renderers/translated_doi_attribute_renderer.rb]: These comments don't seem to apply to this class.

File: app/views/curation_concerns/base/_attribute_rows.html.erb [hunk 2]
```diff
@@ -10,7 +10,7 @@
 <%= presenter.attribute_to_html(:date_created, render_as: :linked, search_field: 'date_created_tesim') %>
 <%= presenter.attribute_to_html(:subject, render_as: :faceted) %>
 <%= presenter.attribute_to_html(:language, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:identifier, render_as: :linked, search_field: 'identifier_tesim') %>
+<%= presenter.attribute_to_html(:identifier, render_as: :translated_doi, search_field: 'identifier_tesim') %>
```
Comment (awead) [inline app/views/curation_concerns/base/_attribute_rows.html.erb:13]: If you're using the `show_fields/_identifier.html.erb` partial below, I'm wondering if this has any effect?

**Review by awead — CHANGES_REQUESTED** [review]

File: app/views/curation_concerns/base/_attribute_rows.html.erb (hunk 2 above)
Comment (carolyncole) [reply app/views/curation_concerns/base/_attribute_rows.html.erb:13]: @awead this is for the work.  The other is for the collection

**Comment by carolyncole** [comment]
@awead I updated the comments.

**Review by awead — APPROVED** [review]

**Pull request merged**